### ISSUE TITLE
Extension

### ISSUE THREAD

**Issue by Msebaie2** [MirnaHamdy/Yajny#42]
I can't send SMS with Extension link as there is no availability to add my number, check below screenshot

https://staging.yajny.enactweb.com/en/install-browser-extension

![image](https://user-images.githubusercontent.com/58039276/71353696-84127600-2582-11ea-93b6-edc029490984.png)

**Comment by enuat** [comment]
All the content is dynamic from blocks..
https://prnt.sc/qenfia

http://admin.staging.yajny.enactweb.com/manage/blocks/49/edit?
http://admin.staging.yajny.enactweb.com/manage/blocks/47/edit?

Whatever button name & link entered it will hyperlink dynamically.

Since, browser extension is not deployed, we've kept dummy text/hyperlink, which will be updated when browser extension is deployed.

For now, I have changed it to demonstrate that it's dynamic.
